src/AppBundle/Entity/Adherent.php

```diff
@@ -101,6 +102,11 @@ class Adherent implements UserInterface, GeocodableInterface
      */
     private $interests;
 
+    /**
+     * @ORM\Column(type="array")
+     */
+    private $emailSubscriptions = [];
```

Comment:
It would have been better to have 3 flags instead of an array... It won't be easy to select adherents who subscribed to a list or another...